docs/configuration.md

```diff
@@ -44,6 +44,8 @@ but overflows could happen in case of slow provisioners and / or higher number o
 concurrent reconciles. For such reasons, it is highly recommended to keep
 BMO_CONCURRENCY value lower than the requested PROVISIONING_LIMIT. Default is 20.
 
+`IRONIC_EXTERNAL_URL` -- This is the URL where Ironic will find the image for the node.
```

Comment:
Let's maybe call it `IRONIC_EXTERNAL_URL_V6` to make it clear it's only used for IPv6?